### PR TITLE
Limiting number of processors for concurrent processing

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

**Comment by khatton-ft** [comment]
:+1:

File: consumer/consumer.go [hunk 1]
```diff
@@ -138,10 +138,34 @@ func (c *DefaultQueueConsumer) consume() ([]Message, error) {
 		return nil, err
 	}
 
-	for _, msg := range msgs {
-		if c.config.ConcurrentProcessing == true {
-			go c.handler(msg)
-		} else {
+	if c.config.ConcurrentProcessing == true {
+		processors := 100
```
Comment (tamas-molnar) [inline consumer/consumer.go]: Consider parameterizing this, with an optional params, with default 100 in case we want to tweak it.

Comment (catalin-ilea) [inline consumer/consumer.go]: Done

**Comment by scott-ace-newton** [comment]
:+1:

**Pull request merged**